src/libslic3r/PerimeterGenerator.cpp

```diff
@@ -1291,6 +1291,29 @@ static ExtrusionEntityCollection traverse_extrusions(const PerimeterGenerator& p
             extrusion_paths_append(paths, *extrusion, role, is_external ? perimeter_generator.ext_perimeter_flow : perimeter_generator.perimeter_flow);
         }
 
+        auto check_and_stagger_path = [perimeter_generator](ExtrusionPath& cur_path) { 
```

Comment:
Based on the codes below, would it be better call it "update_stagger_path" for clarity?